corgi/tasks/brew.py

```diff
@@ -29,7 +29,12 @@ def slow_fetch_brew_build(build_id: int, save_product: bool = True, force_proces
     else:
         logger.info("Fetching brew build with build_id: %s", build_id)
 
-    component = Brew().get_component_data(build_id)
+    try:
+        component = Brew().get_component_data(build_id)
+    except BrewBuildTypeNotSupported as exc:
+        logger.warning(str(exc))
```

Comment:
I'd generally prefer logger.error but other than that nit, LGTM.

Reply:
Why?

Reply:
When logging an exception that got raised, higher log level feels more natural to me. But it's just a personal preference really.